sbc_translation/models/import_letters_history.py

```diff
@@ -0,0 +1,54 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014-2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emmanuel Mathier, Loic Hausammann <loic_hausammann@hotmail.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+"""
+This module reads a zip file containing scans of mail and finds the relation
+between the database and the mail.
+"""
+import logging
+
+from openerp import models, api
+from . import translate_connector
+
+logger = logging.getLogger(__name__)
+
+
+class ImportLettersHistory(models.Model):
+    """
+    Redefine save button when problem to import
+    """
+
+    _inherit = 'import.letters.history'
+
+    @api.multi
+    def button_save(self):
+        """
+        save the import_line as a correspondence
+        """
+        logger.info("BUTTON SAVE ON CHILD !!!")
+        try:
+            super(ImportLettersHistory, self).button_save()
+        except Exception:
+            logger.info("EXCEPTION RAISE DURING IMPORT !!!")
```

Comment:
Can you have something more precise and less dramatic ? Like "Exception during letter import" :wink: